src/ControlSurface/Controls/Control.qml

```diff
@@ -0,0 +1,19 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+
+// Control Skeleton
+Item {
+    id: control
+    // Variables send by score
+    property string controlCustom
```

Comment:
Isn't that duplicated with what's in ScoreColorPoint

Reply:
I wanted to make inherit the controls from control.qml but I did not suceeded

Reply:
so, Controls.qml is useless for the moment